lib/README.md

```diff
@@ -0,0 +1,6 @@
+Source code
+===========
+
+Place here the source code of your app
+
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-sth-comet/commit/9916b12f4cf7eef2d0ef8e80a0791204126cba3e